UWPHook/SteamGridDb/SteamGridDbApi.cs

```diff
@@ -0,0 +1,153 @@
+
+            GameResponse[] games = null;
+            HttpResponseMessage response = await httpClient.GetAsync(path);
+
+            if(response.IsSuccessStatusCode)
+            {
+                var parsedResponse = await response.Content.ReadAsAsync<ResponseWrapper<GameResponse>>();
+                games = parsedResponse.Data;
+            }
+
+            return games;
+        }
+
+        public async Task<GridResponse[]> GetGameGrids(
+            int gameId,  
+            string dimensions = null, 
+            string types = null, 
+            string styles = null,
+            string nsfw = "any", 
+            string humor = "any") 
+        {
+            string path = $"grids/game/{gameId}?";
+
+            if (!String.IsNullOrEmpty(dimensions))
+                path += $"dimensions={dimensions}&";
+
+            if (!String.IsNullOrEmpty(types))
+                path += $"types={types}&";
+
+            if (!String.IsNullOrEmpty(styles))
+                path += $"styles={styles}&";
+
+            if (!String.IsNullOrEmpty(nsfw))
+                path += $"nsfw={nsfw}&";
+
+            if (!String.IsNullOrEmpty(humor))
+                path += $"humor={humor}&";
+
+            GridResponse[] grids = null;
+            HttpResponseMessage response = await httpClient.GetAsync(path);
+
+            if (response.IsSuccessStatusCode)
+            {
+                var parsedResponse = await response.Content.ReadAsAsync<ResponseWrapper<GridResponse>>();
+                grids = parsedResponse.Data;
+            }
+
+            return grids;
+        }
+
+        public async Task<HeroResponse[]> GetGameHeroes(
+            int gameId,
+            string types = null,
+            string dimensions = null, 
+            string styles = null, 
+            string nsfw = "any", 
+            string humor = "any")
+        {
+            string path = $"heroes/game/{gameId}?";
+
+            if (!String.IsNullOrEmpty(dimensions))
+                path += $"dimensions={dimensions}&";
+
+            if (!String.IsNullOrEmpty(types))
+                path += $"types={types}&";
+
+            if (!String.IsNullOrEmpty(styles))
+                path += $"styles={styles}&";
+
+            if (!String.IsNullOrEmpty(nsfw))
+                path += $"nsfw={nsfw}&";
+
+            if (!String.IsNullOrEmpty(humor))
+                path += $"humor={humor}&";
+
+            HeroResponse[] heroes = null;
+            HttpResponseMessage response = await httpClient.GetAsync(path);
+
+            if (response.IsSuccessStatusCode)
+            {
+                var parsedResponse = await response.Content.ReadAsAsync<ResponseWrapper<HeroResponse>>();
+                heroes = parsedResponse.Data;
+            }
+
+            return heroes;
+        }
+
+        public async Task<LogoResponse[]> GetGameLogos(
+            int gameId,  
+            string types = null, 
+            string styles = null,
+            string nsfw = "any", 
```

Comment:
Maybe would be cool to parameterize these, offering a user choice, like the Playnite launcher offers, many people who use SteamGridDB would appreciate this.

Reply:
But you may resolve this and, once merged, i could add these settings!

Reply:
I'll leave this up to you then, since I'm not familiar with Playnite, so you'll be able to organize it better, imo